mrtt-ui/src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react'
+
+export const useLocalStorage = (keyName, defaultValue) => {
+  const [storedValue, setStoredValue] = useState(() => {
+    try {
```

Comment:
why are we using a try/catch here? we're not using an async function or producing any errors from trying to get something from local storage

Reply:
How do you suggest we change this?

Reply:
we can remove the try/catch and just keep the if/else

Reply:
Addressed in commit https://github.com/globalmangrovewatch/gmw-users/pull/157/commits/1359574b10c428886413738bbbee968031ae3ee9